frontend/src/pages/TermPlanner/DraggableCourse/DraggableCourse.jsx

```diff
@@ -67,19 +67,17 @@ const DraggableCourse = ({
           >
             {!isDragDisabled && shouldHaveWarning
               && (errorIsInformational ? <InfoCircleOutlined style={{ color: "#000" }} /> : (
-                <WarningOutlined
-                  style={{ color: "#DC9930", fontSize: "16px" }}
-                />
+                <S.IconWarningOutlined />
               ))}
             <S.CourseLabel>
               {isSmall ? (
-                <Text>{code}</Text>
+                <S.Text>{code}</S.Text>
               ) : (
                 <div>
-                  <Text strong>
-                    {code}
-                  </Text>
-                  <Text>: {title} </Text>
+                  <S.Text>
```

Comment:
Dont think we need to have a styled component for Text here? Could you use `<Text className="text">` here instead?